Add DropDownWidth property for ComboBox control (#536)
telerik/maui-docs: controls/combobox/configuration.md

## DropDown Behavior

> ComboBox provides the following properties for managing the drop down:

Reviewer: ```suggestion
To manage the dropdown of the ComboBox, use the following properties:
```

* `DropDownWidth`(`double`)&mdash;Defines the width of the dropdown of the control.
* `DropDownHeight`(`double`)&mdash;Defines the height of the dropdown of the control.

> * `IsDropDownOpen`(`bool`)&mdash;Defines whether the drop down part of the control is opened. Default value is `true`. 

Reviewer: ```suggestion
* `IsDropDownOpen`(`bool`)&mdash;Defines whether the dropdown part of the control is opened. The default value is `true`. 
```

> * `IsDropdownClosedOnSelection`(`bool`)&mdash;Defines whether the drop down should be closed when item is selected/deselected. The default value is `true`.

Reviewer: ```suggestion
* `IsDropdownClosedOnSelection`(`bool`)&mdash;Defines whether the dropdown will be closed when the item is selected or deselected. The default value is `true`.
```
